apps/script/deploy-storybook.sh

```diff
@@ -46,7 +46,7 @@ if [ -z "$CHANGED_FILES" ]; then
 fi
 
 # Commit the changes to the gh-pages branch
-git config user.name "Travis CI"
+git config user.name "Circle CI"
 git config user.email "paul@code.org"
```

Comment:
Seems like a bad thing that we've got your email address hard-coded here for a deploy step.